cg/meta/upload/scout/balsamic_config_builder.py

```diff
@@ -0,0 +1,57 @@
+import logging
+
+from housekeeper.store import models as hk_models
+
+from cg.apps.lims import LimsAPI
+from cg.constants.scout_upload import BALSAMIC_CASE_TAGS, BALSAMIC_SAMPLE_TAGS
+from cg.meta.upload.scout.hk_tags import CaseTags, SampleTags
+from cg.meta.upload.scout.scout_config_builder import ScoutConfigBuilder
+from cg.meta.upload.scout.scout_load_config import BalsamicLoadConfig, ScoutBalsamicIndividual
+from cg.meta.workflow.balsamic import BalsamicAnalysisAPI
+from cg.store import models
+
+LOG = logging.getLogger(__name__)
+
+
+class BalsamicConfigBuilder(ScoutConfigBuilder):
+    def __init__(
+        self, hk_version_obj: hk_models.Version, analysis_obj: models.Analysis, lims_api: LimsAPI
+    ):
+        super().__init__(
+            hk_version_obj=hk_version_obj, analysis_obj=analysis_obj, lims_api=lims_api
+        )
+        self.case_tags: CaseTags = CaseTags(**BALSAMIC_CASE_TAGS)
+        self.sample_tags: SampleTags = SampleTags(**BALSAMIC_SAMPLE_TAGS)
+        self.load_config: BalsamicLoadConfig = BalsamicLoadConfig(track="cancer")
+
+    def include_case_files(self):
+        LOG.info("Including BALSAMIC specific case level files")
+        self.load_config.vcf_cancer = self.fetch_file_from_hk(self.case_tags.snv_vcf)
+        self.load_config.vcf_cancer_sv = self.fetch_file_from_hk(self.case_tags.sv_vcf)
+        self.include_multiqc_report()
+
+    def include_sample_files(self, sample: ScoutBalsamicIndividual):
+        LOG.info("Including BALSAMIC specific sample level files")
+
+    def build_config_sample(self, sample_obj: models.FamilySample) -> ScoutBalsamicIndividual:
+        """Build a sample with balsamic specific information"""
+        sample = ScoutBalsamicIndividual()
```

Comment:
could you use a more verbose variable name for this sample and the sample in the method right below? Sure, they are not sharing scope and are referring to different objects, but still it might be a bit confusing to follow on Monday mornings :P

Reply:
Haha yes, fixed